packages/neo-one-server-client/src/ServerManager.ts

```diff
@@ -100,15 +116,24 @@ export class ServerManager {
 
     if (pid !== undefined) {
       if (isRunning(pid)) {
-        const sameVersion = await isSameVersion({
+        const serverVersion = await getRelativeServerVersion({
           port,
           expectedVersion: this.serverVersion,
         });
+        const clientVersion = 0;
 
-        if (sameVersion) {
+        if (serverVersion === clientVersion) {
           return pid;
         }
 
+        if (serverVersion === undefined) {
+          return undefined;
+        }
+
+        if (serverVersion > clientVersion) {
+          throw new Error('[NEO-ONE] version conflict: server is newer than client');
```

Comment:
Let's clearly instruct the user to upgrade. 
```
throw new Error('[NEO•ONE] version conflict: server is newer than client. Please upgrade NEO•ONE and try again.')
```

Let's also open a card to add it somewhere in the documentation and then put a link to the docs here.

Reply:
👍 Docs Card created: 
https://github.com/neo-one-suite/neo-one/issues/764